### PR TITLE
Bump PHPStan analysis to level 6

### PR DESCRIPTION
**Pull request by antograssiot**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |
| License       | MIT
| Doc PR        |

This PR will allow to bump PHPStan level on master too once #2272 will be merged.

### CODE REVIEW

File: tests/Metadata/Extractor/ExtractorTestCase.php [hunk 1]
```diff
@@ -175,7 +175,7 @@
                 'attributes' => null,
                 'properties' => null,
             ],
-            \ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\DummyBis::class => [
+            '\ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\DummyBis' => [
```
[the remark this answers is not in the document]
Comment (antograssiot) [reply tests/Metadata/Extractor/ExtractorTestCase.php]: The class doesn't exist @soyuka so we can'nt use it

[the remark this answers is not in the document]
Comment (teohhanhui) [reply tests/Metadata/Extractor/ExtractorTestCase.php]: Actually, the class does not need to exist.

"magic" :crystal_ball:

[the remark this answers is not in the document]
Comment (teohhanhui) [reply tests/Metadata/Extractor/ExtractorTestCase.php]: `use` does not trigger autoloading. :smile:

[the remark this answers is not in the document]
Comment (antograssiot) [reply tests/Metadata/Extractor/ExtractorTestCase.php]: Ok @teohhanhui but I still find it weird to use it instaed of a string and to add a rule in phpstan to fix this.... I can revert this part, not an issue

[the remark this answers is not in the document]
Comment (teohhanhui) [reply tests/Metadata/Extractor/ExtractorTestCase.php]: I actually prefer that we use a string too! And better with an obviously made-up FQCN...

[the remark this answers is not in the document]
Comment (soyuka) [reply tests/Metadata/Extractor/ExtractorTestCase.php]: Sure mb, let's use a class :).

**Comment by dunglas** [comment]
Can you rebase please @antograssiot?

**Comment by antograssiot** [comment]
rebased @dunglas, it should be all green soon hopefully

**Comment by soyuka** [comment]
tyvm @antograssiot !

**Comment by antograssiot** [comment]
My pleasure. Thanks for reviewing and merging my PRs (globaly I mean, not only this one)

**Pull request merged**